### PR TITLE
added 8 new exercises

### PR DESCRIPTION
**Pull request by kapitaali**
Hi, 

this is my first pull request ever, so if anything goes wrong, that might be the reason.

WHAT HAS BEEN DONE

I have added 8 new exercises to COBOL track: acronym, all-your-base, atbash-cipherr, binary, circular-buffer, complex-numbers, matching-brackets, pascals-triangle. 

These exercises have been solved by me, solutions are found in the respective proof.ci.cob files.

WHAT IS MISSING

I could have appreciated some sort of guidelines as to what is expected when doing a pull request, like if you would need just the .cob and .cut files (which the pull request was about to contain before I realized that there were hidden directories with solutions) or do you need a pull request submitter to solve the exercises. I tried to look around and was not able to find anything. Luckily I did them myself.

### CODE REVIEW

**Comment by ErikSchierboom** [comment]
@kapitaali Thanks for the PR! It's quite a big PR, so it might take a little while for me to review. I'll come back to this later.

**Comment by axtens** [comment]
Also reviewing. My first reaction to Acronym is that while the proof.ci.cob may work, it's not particularly idiomatic cobol. If acronym were already a task, then the provided solution would do as a first attempt. I'd then encourage the student to look into better string length routines, use of indexing into strings etc.

**Comment by axtens** [comment]
It does, though, raise the question: is the proof.ci.cob supposed to be "good" cobol or just something-that-works?

**Comment by ErikSchierboom** [comment]
> just something-that-works?

It's this

**Comment by ErikSchierboom** [comment]
Wonderful work! There's a couple of things that need to be fixed before this can be merged:

1. Fix the [configlet issue](https://github.com/exercism/cobol/actions/runs/6729910632/job/18291622387). All of the newly added exercises need to have an entry in the root `config.json` file's `exercises.practice` array: https://github.com/exercism/cobol/blob/main/config.json#L29 See https://exercism.org/docs/building/tracks/config-json#h-practice-exercises for more information.
2. CI needs to be fixed: https://github.com/exercism/cobol/actions/runs/6729910587/job/18291621996?pr=105 It looks like not all of the proofs pass all the tests.

Once these things have been fixed, we can merge this.

Side note: if you'd like to add more exercises after this, I would suggest doing one PR per exercise, as that makes reviewing easier and increases the chance of the PR being merged sooner.

**Comment by ErikSchierboom** [comment]
Just one more configlet error:

> JSON parsing error:
/home/runner/work/cobol/cobol/exercises/practice/all-your-base/.meta/config.json(21, 1) Error: found trailing comma. } or key/value pair expected

**Comment by kapitaali** [comment]
argh :D

File: config.json [hunk 1]
```diff
@@ -147,7 +147,7 @@
         "practices": [],
         "prerequisites": [],
         "difficulty": 6
-      },
+      },
```
Comment (ErikSchierboom) [inline config.json:150]:
```suggestion
      },
```

**Review by ErikSchierboom — APPROVED** [review]
You happy with this @axtens?

**Review by axtens — APPROVED** [review]
I am very happy to see the track expanded in this way. I agree with @ErikSchierboom that submitting extra tasks individually does help with the approval process. Apart from that, approved!

**Comment by ErikSchierboom** [comment]
Merged! Thanks

**Comment by iHiD** [comment]
@kapitaali Thank you :)

**Pull request merged**